include/maths/CLocalOutlierFactors.h

```diff
@@ -51,8 +51,10 @@ class MATHS_EXPORT CLocalOutlierFactors {
     //! the number of nearest neighbours to use based on the data
     //! characteristics.
     template<typename POINT>
-    static void normalizedLof(const std::vector<POINT>& points, TDoubleVec& scores) {
-        normalizedLof(numberNeighbours(E_Lof, points), project(points), points, scores);
+    static void normalizedLof(std::vector<POINT> points, TDoubleVec& scores) {
```

Comment:
if you want to move points, shouldn't this be `std::vector<POINT>&& points` ?

Reply:
The reason is I want to call:
```
CLocalOutlierFactors::normalizedLof(std::move(mypoints), scores);
```
in code running outlier detection on some collection `mypoints` derived from a data frame, which I don't subsequently need to preserve in the calling context. This will move the container into this local variable (very nearly as efficient), which is later on moved into an annotated collection. However, there seems no reason to force that the calling code always to have to move.